.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 2.1
+orbs:
+  go: circleci/go@1.7.1
+
+jobs:
+  test:
+    parameters:
+      version:
+        default: "current"
```

Comment:
Looking at [cimg/go](https://circleci.com/developer/images/image/cimg/go) there doesn't appear to be a "current" tag. In that case, we should probably be hard-coding this.

Reply:
Discussed some details offline and fixed.